.github/workflows/reusable-ubuntu.yml

```
@@ -4,9 +4,11 @@ on:
       config_hash:
         required: true
         type: string
-      options:
-        required: true
-        type: string
+      free-threading:
+        description: Whether to use no-GIL mode
```

Let's avoid the forbidden term :)
```suggestion
        description: Whether to use free-threaded mode
```

---

It's forbidden? :o

---

Yes, see https://github.com/python/devguide/issues/1353.

---

Thanks for the reminder, I missed it here.

---

@hugovk FYI the buildbots still use no-GIL terminology: https://buildbot.python.org/#/builders/1258/builds/2403